docs/proposals/network-policy.md

```diff
@@ -197,6 +254,35 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString `json:"port,omitempty"`
 }
 
+type NetworkCIDRRange struct {
+	// QUESTION: we could just do "Start" and "End" without a prefix to
+	// specify the range.  While this provides a simpler structure, it may
+	// be more error-prone since its less easily validated, and isn't very
+	// nice for IPv6 addresses.
```

Comment:
Basically, Prefix makes sense if you want to deny traffic by default and only allow to a few networks, but not if you want to allow by default and deny to a few networks:

"Allow only to 10.0.0.0/8 and 192.168.42.0/24":
- Start: 10.0.0.0, Prefix: 8
- Start: 192.168.42.0, Prefix: 24

"Allow everywhere except 10.0.0.0/8 and 192.168.42.0/24":
- Start: 0.0.0.0, Prefix: 32, End: 9.255.255.255
- Start: 11.0.0.0, Prefix: 32, End: 192.168.41.255
- Start: 192.168.43.0, Prefix: 32, End: 255.255.255.255